src/util/functions.js

```diff
@@ -1,5 +1,5 @@
 import distanceInWordsStrict from 'date-fns/distance_in_words_strict';
-const CIVIC_API_KEY = process.env.REACT_NATIVE_API_KEY;
+const REACT_NATIVE_API_KEY = process.env.REACT_NATIVE_API_KEY;
```

Comment:
If you were having problems with the district match I think making a different alias for the api key is unlikely to solve that

Reply:
Haha. My issue turned out not to be related to API keys.
This name change was more for consistency since there should be no CIVIC_API_KEY in the .env files and REACT_NATIVE_API_KEY is the google key that Google Civic and any other APIs can use AFAIK